### DISCUSSION TITLE
Creating Wordpress Like Taxonomies

### DISCUSSION THREAD

**Discussion by ghost** [strapi/strapi#7761] [Legacy Topics]
In Wordpress it is possible to create custom taxonomies that relate to a particular post type.  So, for instance, you can have two content types (Blogs and Portfolios) which each have their own proprietary `category` taxonomy.  

I am wondering how to create something similar in Strapi.  In Strapi it is easy to create content types and to associate different content types.  So, for instance, I can have a `Posts` content type and a `Categories` content type and associate the two of them with a relations field.  But what do I do if I now want to have a `Categories` content type for `Portfolios`?  Should I create a `PortfoliosCategories` content type and relate that to `Portfolios`?  That seems to work, but does not seem so elegant.

Should I use relate the original `Categories` content type (the same one that I use for the `Posts` content type)?  That does not seem so smart.

The other alternative is to to create some sort of sub-content type (i.e., a taxonomy) that relates solely the parent content type.  Is that possible with Strapi -- if so how?

**Comment by derrickmehaffy** [comment]
Your original idea was the proper one, something like `PortfoliosCategories`, though ugly, is the best case here. We don't allow content-types with the same name, nor locking specific content-types to another.

**Reply by ghost** [reply]
Okay - I hear.

With that said, I think something like the Wordpress taxonomy structure would be a nice addition at some point in the future.

Either way, thanks for you help.
